src/clientStub.ts

```diff
@@ -55,7 +55,7 @@ export class DgraphClientStub {
         }
 
         this.options = options;
-
+        this.options.header = {};
```

Comment:
this would erase `options.headers` if it exists.

```
if (this.options.headers === undefined) {
 this.options.headers = {};
}
```
would be much safer and backwards compatible

Something that clients may rely on, e.g. [Ratel](https://github.com/dgraph-io/ratel/blob/master/client/src/lib/helpers.js#L79)